ui/src/app/components/demographic-survey-v2.tsx

```diff
@@ -663,21 +670,27 @@ export const DemographicSurvey = fp.flow(withProfileErrorModal)(
               {
                 label: 'Prefer not to answer',
                 value: EthnicCategory.PREFERNOTTOANSWER,
-                onChange: (checked) =>
+                onChange: (checked) => {
                   onUpdate(
                     'ethnicCategories',
                     checked ? [EthnicCategory.PREFERNOTTOANSWER] : []
-                  ),
+                  );
+                  setShowAiAnOptions(!checked);
+                  setShowAsianOptions(!checked);
+                  setShowBlackOptions(!checked);
+                  setShowHispanicOptions(!checked);
+                  setShowMeNaOptions(!checked);
+                  setShowNhPiOptions(!checked);
+                  setShowWhiteOptions(!checked);
+                },
               },
             ]}
             multiple
             selected={survey.ethnicCategories}
             onChange={(value) =>
               onUpdate(
                 'ethnicCategories',
-                value.length > 1
-                  ? value.filter((v) => v !== EthnicCategory.PREFERNOTTOANSWER)
-                  : value
+                value.filter((v) => v !== EthnicCategory.PREFERNOTTOANSWER)
```

Comment:
This was a little tricky. In the end, no, it should not change behavior, because the question onChange is fired and then the option onChange is fired. In the case that just PREFERNOTTOANSWER is checked (with no options selected before hand), an intermediary value will be different, but the end result will still be the same. I can walk you through it if you would like.